src/AWS.Deploy.Common/ProjectDefinitionParser.cs

```diff
@@ -37,6 +37,7 @@ public async Task<ProjectDefinition> Parse(string projectPath)
         {
```

Comment:
As you are updating this code, can you add documentation on the behavior of `projectPath` param?

Reply:
Added